examples/prom/maxwell_local_rom_greedy.cpp

```diff
@@ -0,0 +1,725 @@
+/******************************************************************************
+ *
+ * Copyright (c) 2013-2023, Lawrence Livermore National Security, LLC
+ * and other libROM project developers. See the top-level COPYRIGHT
+ * file for details.
+ *
+ * SPDX-License-Identifier: (Apache-2.0 OR MIT)
+ *
+ *****************************************************************************/
+
+//               libROM MFEM Example: parametric local ROM for Maxwell equation (adapted from ex3p.cpp)
+//
+// Description:  This example code uses MFEM and librom to define a projection-based
+//               reduced-order model for a simple electromagnetic diffusion
+//               problem corresponding to the second order definite Maxwell
+//               equation: curl curl E + E = f. The boundary condition is given by
+//               E x n = <given tangential field>. Here, we use a given exact
+//               solution E and compute the corresponding r.h.s. f. Each component
+//               of the vector field E is assumed to be a sin wave, with frequency
+//               controlled by the parameter kappa.
+//
+//               The example highlights the greedy algorithm. The build_database phase
+//               builds a global ROM database using different frequncies and a latin-hypercube
+//               sampling procedure. The use_database phase uses the global ROM database,
+//               builds the ROM operator, solves thereduced order system, and
+//               lifts the solution to the full order space.
+//
+// build_database phase: maxwell_local_rom_greedy -build_database -greedy-param-min 1.0 -greedy-param-max 1.2 -greedy-param-size 5 -greedysubsize 2 -greedyconvsize 3 -greedyrelerrortol 0.01
+// use_database phase:   maxwell_local_rom_greedy -fom -f 1.15 (create a new solution to compare with)
```

Comment:
Consider changing the description to `FOM phase`? Maybe do the same for the Poisson example too.

Reply:
Good point, I changed the description here and also for the larger example.